### DISCUSSION TITLE
Regarding 1920x1080 fix

### DISCUSSION THREAD

**Discussion by Legacy-TacticalGamingInteractive** [citizenfx/fivem#1631] [Engineering Discussion]
https://github.com/citizenfx/fivem/commit/29fc26752ab72805f53a2c489fc46094ca1f8714

How would a server owner go about enabling this feature?

**Comment by blattersturm** [comment]
A server owner wouldn't enable this feature. A server owner would fix their resources to work across resolutions by correctly using relative units, or push the authors of their resources to do so. 

This is a feature for players to make playing on servers with owners that don't care bearable. Not a feature to allow server owners to be lazy.
